src/main/groovy/com/avast/gradle/dockercompose/ComposeSettings.groovy

```diff
@@ -89,7 +89,7 @@ abstract class ComposeSettings {
 
     @Inject
     ComposeSettings(Project project, String name = '', String parentName = '') {
-        this.nestedName = parentName + name
+        this.nestedName = (parentName + name).toLowerCase()
         this.safeProjectNamePrefix = generateSafeProjectNamePrefix(project)
```

Comment:
IMHO we should lowercase also this value 🤔 

The question then is if it were better to move the lowercasing to the `projectNameProvider` 🤔

Reply:
`generateSafeProjectNamePrefix` generates only lowercase value as far as I know. 

But I think it may actually be a good idea to just lowercase whatever the provider returns. That would also include to lowercase `customProjectName`

Reply:
That could of course again be a problem from user view, because then `projectName` would differ to the input `projectName`

Reply:
Agree 👍

Reply:
Okay how about 5b5df1ab20a1506fc5d3f82bac400f614053eee7